Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM decred/decred-golang-builder-1.9 as builder
+COPY . /go/src/github.com/decred/dcrwallet
+WORKDIR /go/src/github.com/decred/dcrwallet
+RUN dep ensure
```

Comment:
I'm not a golang dev so I could be wrong, but following the #dev channel in Slack leads me to believe that the project doesn't use `dep` any more